packages/node-sdk/paima-db/migrations/up.sql

```diff
@@ -193,4 +193,18 @@ CREATE TABLE cde_cardano_asset_utxos (
   policy_id TEXT NOT NULL,
   asset_name TEXT NOT NULL,
   PRIMARY KEY(cde_id,tx_id,output_index,cip14_fingerprint)
-);
\ No newline at end of file
+);
+
+CREATE TABLE cde_tracking_cardano_pagination (
+  cde_id INTEGER PRIMARY KEY,
+  cursor TEXT NOT NULL,
+  finished BOOLEAN NOT NULL
+);
+
+CREATE TABLE cde_cardano_transfer (
```

Comment:
I'm actually starting to wonder if it does actually make sense to store this stuff... There are currently no utility function using this, and currently I'm not sure why would you want to do something with this.

Reply:
As in you want to pass this info to the state machine without actually storing it to longer term storage?

Reply:
Well, the question can also be reversed into: "What util functions do we need here?". Would a game ever want to fetch the transaction by id? For example. 

Currently they may need to because the actual rax tx is not in the concise repr, but we can also add the rawTx there. But you could also just go to the db directly.

Or could you want to get all the transactions for a certain address? Or the latest? Because in this case we should also have a table indexing the txs by input address, for example.